src/components/VueTable/VueTable.ts

```diff
@@ -289,15 +301,16 @@ export default class VueTable extends Vue {
     }
   }
 
-  updateSelectedCell(header: string, row: number, col: any) {
+  updateSelectedCell(header: string, rowIndex: number, colIndex: any) {
     if (!this.setFirstCell) {
-      this.$set(this.tbodyData[row][header], 'rectangleSelection', true);
+      this.$set(this.tbodyData[rowIndex][header], 'rectangleSelection', true);
       this.setFirstCell = true;
     }
+    this.highlightTdAndThead(rowIndex, colIndex);
```

Comment:
Move this line at the end of the function and add a comment which explain the function does